source/agora/common/BitMask.d

```diff
@@ -77,11 +77,11 @@ public struct BitMask
         this.bytes[] = bytes;
     }
 
-    // set the bits that are set in given BitMask
+    // set the bits that are set in given BitMask (this is NOT a bitwise `OR`)
     public auto opOpAssign (string op : "|") (in BitMask add)
     {
         assert(this.length == add.length, "BitMask assignment must be for same bit length");
-        this.bytes[] |= add.bytes[];
+        this.bytes[] = add.bytes[];
```

Comment:
That's just a plain `opAssign`

Reply:
I did try that but it had unwanted side effects.

Reply:
I will retry with `opAssign` as I agree it is a bit odd now.

Reply:
Ended up function `copyFrom` for the reasons explained in the commit description.